### ISSUE TITLE
Scaling impressions

### ISSUE THREAD

**Issue by warius1983** [datchannin/Brothers-WoW#89]
Ok I gave a first look on all abilities, for sure we have in general a nerf before this new system and this, this is not bad at all.

However, I am having some troble in creating some builds which are not too much glasscannon, or with not much fatique.

Essentially I have been trafficing a little with some mods and files, Now I could suggest you 3 solutions to the problems:

a)
![attr](https://user-images.githubusercontent.com/73758055/168489234-318b6713-dcaf-4ebe-a820-bde2733370c7.png)

I increased the value of statistics on every level, ok numers are way too big, But from character.nut I increased the value of every statistic for level up, essentially I'd say to increase health or fatique by 6-8 + stars talents, as I said time ago, this way you can invest almost always in mastery and in other stats without losing too much behind

b) 
![image](https://user-images.githubusercontent.com/73758055/168489600-4c5b3c33-a443-4717-a08b-52a5631deed9.png)

4 upgrades per level instead of the usual 3, i used this mod and works with yours https://www.nexusmods.com/battlebrothers/mods/381?tab=description

c)
Increase the possible value for every class during level ups, I tried changing values in the single backgrounds.nut files but it didn't work.

I think solution C could be the best, but nonetheless i will try again something different, scaling comments will be done after a whole adventure.

**Comment by datchannin** [comment]
New parameters were added to give **possibility** for using. If I add additional level up or big bonus - it will be the same as level auto scaling. Because now player should decide what he want increase: fatigue or spell mastery, or hitpoints or spell mastery.
It player has 4 level ups, he will increase spell mastery every level upp.  

Let's try get example.
1 variant (3 level upps). You should choose between spell mastery, hp, fatigue and ranged defense.
10 levels. 4 points from level up. Average:
10 levels * 3 level upps = 30 level upps. Divide them between 4 parameters = 7-8 level upps for every parameter.
7.5 level upps * 4 points from level up = 30 points for every parameter.
So result will be:
spell mastery = 30
hp = 30
fatigue = 30
ranged defence = 30

2 variant (4 level upps). You should choose between spell mastery, hp, fatigue and ranged defense.
10 levels. 4 points from level up. Average:
10 levels * 4 level upps = 40 level upps. Divide them between 4 parameters = 10 level upps for every parameter.
10 level upps * 4 points from level up = 40 points for every parameter.
So result will be:
spell mastery = 40
hp = 40
fatigue = 40
ranged defence = 40

So if in first example try to distribute parameters without spell mastery:
spell mastery = 0
hp = 30 + 10 = 40
fatigue = 30 + 10 = 40
ranged defence = 30 + 10 = 40
This will be the same as give auto 40 points to spell mastery. And player will not should do any choose. Only adds +x points for mastery every level up. I don't see any difference between auto-scaling from level.

May be player should decide what is better - to have more damage from spells and use recover one more time per combat, or do less damage and take some other perk instead of recover.
What is better - to give more hitpoints for Priest, or spend all upps to mastery and use triple Repair spell per turn but protect priest by additional healer/tank to save him from burst damage.  
If only give +40-60 auto mastery for every character, this parameter will mean nothing.

**Comment by warius1983** [comment]
I agree on four level upgrades, which can be a fail, ok i will try to see some more scripts and testing
